server/config/config.go

```diff
@@ -751,6 +756,7 @@ type ScheduleConfig struct {
 	// overwrite the auto-tuned value by pd-ctl, when the value
 	// is overwritten, the value is fixed until it is deleted.
 	// Default: manual
+	// WARN: StoreLimitMode is deprecated.
 	StoreLimitMode string `toml:"store-limit-mode" json:"store-limit-mode"`
```

Comment:
why not delete it?

Reply:
IMO, we need to mark it as deprecated for about 2 major versions, then remove it totally.

Reply:
I'm going to delete it in the next PR.